webapps/ab-testing-design/body.html

```diff
@@ -0,0 +1,452 @@
+                            If you only want to determine if there is any difference between the two groups, you
+                            should use a
+                            two-tailed test. It means that you are testing both for positive and negative
+                            differences.
+                        </div>
+                        <div class="explanation_text">
+                            However, if you only test in one direction, to find out for instance, whether the
+                            conversion rate is
+                            higher
+                            for B, you may want to use a one-tailed distribution. For example, if you test a new
+                            email template,
+                            your major concern is whether it leads to more conversion.
+                            A two-tailed test is not necessary since you are only interested in positive
+                            changes.
+                        </div>
+                    </div>
+                </span>
+                <select name="tail" id="tail">
+                    <option value="false" class="alert">False</option>
+                    <option value="true">True</option>
+                </select>
+            </div>
+        </div>
+        <div class="text-center">
+            <a id="submit_button" type="submit" class="btn btn-outline-info btn-rounded waves-effect  btn-lg">
+                Compute size</a>
+        </div>
+    </form>
+</div>
+</div>
+</div>
+
+<div class="container">
+    <div class="row">
+        <div class="col-xl">
+            <h2> Sample size </h2>
+            <hr>
+            </hr>
+        </div>
+    </div>
+</div>
+</div>
+
+
+<div class="container" id="less_margin">
+    <div class="row justify-content-md-center">
+        <div class="col-md-auto">
+            <table>
+                <thead>
+                    <tr>
+                        <th id="A_header">A</th>
+                        <th id="B_header">B</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td id="sample_size_A">1085</td>
+                        <td id="sample_size_B">1085</td>
+                    </tr>
+                </tbody>
+            </table>
+            <div class="text-center">
+                <div id="duration" class="d-none">
+                    <div>
+                        This experiment should run for
+                    </div>
+                    <span id="nb_of_days">
+                    </span>
+                    <div>
+                        days
+                    </div>
+                </div> 
+            </div>
+            <div class="text-center">
+                <a id="attribution_button" type="button" class="btn btn-outline-info btn-rounded waves-effect  btn-lg">
+                    Set up an A/B attribution recipe</a>
```

Comment:
Now that we save the params in a folder, the wording should be updated too

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/de02b75af3fbf4b244e688c7c8603c4666e24b47